src/main/java/com/thealgorithms/maths/Means.java

```diff
@@ -0,0 +1,41 @@
+package com.thealgorithms.maths;
+
+import java.util.List;
+
+/**
+ * https://en.wikipedia.org/wiki/Mean
+ * <p>
+ * by: Punit Patel
+ */
+public final class Means {
+
+    private Means() {
+    }
+
+    /**
+     * Arithmetic / Pythagorean Mean = (x₁ + x₂ + ... +  xₙ) / n
+     */
```

Comment:
I would avoid formulas in the doc-strings. What do you think about the following:
```suggestion
    /**
     * @brief computes the [arithmetic mean](https://en.wikipedia.org/wiki/Arithmetic_mean) of the input
     * @param numbers the input numbers
     * @throws IllegalArgumentException empty input
     * @return the arithmetic mean of the input numbers
     */
```